Gemfile.lock

```diff
@@ -48,32 +47,33 @@ GEM
     method_source (0.9.2)
     mini_portile2 (2.4.0)
     minitest (5.11.3)
-    money (6.13.2)
+    money (6.13.4)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.1)
+    nokogiri (1.10.4)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.6)
+    rack (2.0.7)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.0.4)
+    rails-html-sanitizer (1.2.0)
```

Comment:
### [rails-html-sanitizer](https://github.com/rails/rails-html-sanitizer)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.0.4 → 1.2.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **26** commits. See the full changes on [the compare page](https://github.com/rails/rails-html-sanitizer/compare/v1.0.4...v1.2.0).

These are the first 10 commits:
- (4ccc774) [[CI] Test against Ruby 2.5](https://github.com/rails/rails-html-sanitizer/commit/4ccc7742d8cbd95e620887d60b802a1d9a83d424)
- (e4e6cab) [[CI] Allow failure with ruby head](https://github.com/rails/rails-html-sanitizer/commit/e4e6cabeafa2a032e24f06e251d127d10d2cf9fa)
- (fc5fd28) [Merge pull request #71 from nicolasleger/patch-1](https://github.com/rails/rails-html-sanitizer/commit/fc5fd28cc74ad525243f4fe464e8089c99db7d83)
- (d4d823c) [[ci skip] Please don't send more PRs trying to bump Loofah.](https://github.com/rails/rails-html-sanitizer/commit/d4d823c617fdd0064956047f7fbf23fff305a69b)
- (cba410f) [Fix Nokogiri link in documentation](https://github.com/rails/rails-html-sanitizer/commit/cba410ff71d2d0bdb03b59a6bca1d6222b3d0b4d)
- (f82bfd2) [Merge pull request #86 from tebs/fix-documentation-link](https://github.com/rails/rails-html-sanitizer/commit/f82bfd25c02f806e51b5e2ae62334bba4e908ad8)
- (89ae177) [Use a inclusive Bundler version](https://github.com/rails/rails-html-sanitizer/commit/89ae177f05f9336fd7af61b01e657404d4d1f60e)
- (630d2f7) [Update Ruby version matrix on CI](https://github.com/rails/rails-html-sanitizer/commit/630d2f7806067037c16366106662e69ca7b25ade)
- (ccb51ce) [Merge pull request #88 from JuanitoFatas/jf.relax-bundler-de…](https://github.com/rails/rails-html-sanitizer/commit/ccb51ce89d43678f29cb86cb6e1dcf850b8af97f)
- (7d30b71) [Merge pull request #89 from JuanitoFatas/rubies](https://github.com/rails/rails-html-sanitizer/commit/7d30b71eec215229d3e28d23eac6818a7dc581ae)


</details>